src/test/java/core/basesyntax/service/impl/CsvReaderServiceTest.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertThrows;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class CsvReaderServiceTest {
+    private CsvReaderService csvReaderService;
+
+    @BeforeEach
+    void setUp() {
+        csvReaderService = new CsvReaderService();
+    }
+
+    @Test
+    void read_file_ok() {
+        csvReaderService.readFromFile("src/main/resources/input.csv");
+    }
```

Comment:
we need to check result of reading, it easily can return empty string or any other value that do not correspond file content